src/main/java/domain/CarRace.java

```diff
@@ -0,0 +1,62 @@
+package domain;
+
+import exception.IllegalRandomVariableException;
+import view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+import static view.OutputView.printCarsInfo;
+
+public class CarRace {
+
+    private static List<Car> cars;
+    private int tryCount;
+
+    public CarRace(String[] names, int tryCount) {
+        cars = new ArrayList<>();
+        for(int i=0;i<3;i++) cars.add(new Car(names[i]));
```

Comment:
```suggestion
        for (int i = 0; i < 3; i++) {
            cars.add(new Car(names[i]));
        }
```

사소한 부분일지라도 컨벤션 통일에 신경 써주세요!

가독성을 해치는 부분이 될 수 있습니다.

협업하는 동료를 위해서라도 신경쓰면 좋겠죠? ㅎㅎ

이런 것들을 모두 신경쓰시기 어렵다면, 인텔리제이 자동정렬 기능을 활용하셔도 좋아요!